distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -585,5 +590,34 @@ private String createRegistryStoreEntry(String id, String regId, String remoteRe
                 new HashSet(destinations));
 
     }
+
+    private void waitForMockServer() throws Exception {
+        long startTime = System.currentTimeMillis();
+        long timeout = TimeUnit.MINUTES.toMillis(2);
+        boolean available = false;
+        LOGGER.info("Waiting for Csw Mock Server to come up");
+        while (!available) {
+            try {
+                given().auth()
+                        .preemptive()
+                        .basic(ADMIN, ADMIN)
+                        .body("<csw:GetCapabilities service=\"CSW\" xmlns:csw=\"http://www.opengis.net/cat/csw\" xmlns:ows=\"http://www.opengis.net/ows\"/>")
+                        .header("Content-Type", "text/xml")
+                        .expect()
+                        .log()
+                        .all()
+                        .statusCode(200)
+                        .when()
+                        .post(CSW_PATH.getUrl());
+                available = true;
+                LOGGER.info("Csw Mock Server is running");
+            } catch (Error e) {
+                if (System.currentTimeMillis() - startTime > timeout) {
+                    fail("Csw mock server didn't come up within allotted time.");
+                }
+                Thread.sleep(2000);
```

Comment:
Changed to a constant. The amount of time is in line with other itests but still somewhat arbitrary. Lots of tests use 1 second but i chose 2 here so that there wouldn't be as much churn on slower systems.